script/test

```diff
@@ -229,6 +229,70 @@ do_cert_suite()
     fi
 }
 
+do_cert_bbr()
+{
+    if [[ ${THREAD_VERSION} != "1.2" ]]; then
+        echo "cert_bbr only work with THREAD_VERSION=1.2!"
+        exit 1
+    fi
+
+    if [[ ${VIRTUAL_TIME} != "0" ]]; then
+        echo "cert_bbr only work with VIRTUAL_TIME=0!"
+        exit 1
+    fi
+
+    export top_builddir="${OT_BUILDDIR}/cmake/openthread-simulation-1.2"
+    export top_builddir_1_2_bbr="${OT_BUILDDIR}/cmake/openthread-simulation-1.2-bbr"
+    export top_builddir_1_1="${OT_BUILDDIR}/cmake/openthread-simulation-1.1"
+    export PYTHONPATH=tests/scripts/thread-cert
+
+    python3 tests/scripts/thread-cert/run_bbr_tests.py --multiply "${MULTIPLY:-1}" "$@"
+}
+
+do_get_thread_wireshark()
+{
+    echo "Downloading thread-wireshark from https://github.com/openthread/wireshark/releases ..."
+    local download_url=https://github.com/openthread/wireshark/releases/download/ot-pktverify-20200727/thread-wireshark.tar.gz
+    local save_file=/tmp/thread-wireshark.tar.gz
+
+    rm -rf /tmp/thread-wireshark || true
+    rm -rf "${save_file}" || true
+    curl -L "${download_url}" -o "${save_file}"
+    tar -C /tmp -xvzf "${save_file}"
+
+    LD_LIBRARY_PATH=/tmp/thread-wireshark /tmp/thread-wireshark/tshark -v
+    LD_LIBRARY_PATH=/tmp/thread-wireshark /tmp/thread-wireshark/dumpcap -v
+    rm -rf "${save_file}"
+}
+
+do_build_otbr_docker()
+{
+    echo "Building OTBR Docker ..."
+    local otdir
+    local otbrdir
+    local otbr_options="-DOT_THREAD_VERSION=1.2 -DOT_BACKBONE_ROUTER=ON -DOT_DUA=ON -DOT_MLR=ON -DOT_REFERENCE_DEVICE=ON -DOT_SERVICE=ON -DOT_COVERAGE=ON"
```

Comment:
https://github.com/simonlingoogle/ot-br-posix/blob/2c4be87c1fd4baa3dbf69bb3664ee10a330f76fc/CMakeLists.txt#L95-L102

enable `OTBR_BACKBONE_ROUTER` and the three relative switches could be skipped?